tutorials/cloudsdk-js-generator/cloudsdk-js-generator.md

```diff
@@ -71,7 +72,11 @@ Only the input and output directory are mandatory:
 sap-cloud-sdk generate-odata-client -i service-specification/ -o odata-client/
 ```
 
-Note that the generator relies on the `@sap-cloud-sdk/generator` package. If you have not installed this, on the first run a prompt will appear asking you to install it.
+Note that the generator relies on the `@sap-cloud-sdk/generator` package. If you have not installed this, on the first run a prompt will appear asking you to install it. The question looks like "Do you want to install the @sap-cloud-sdk/generator globally?". Choose "no" as it will install the `latest` version of the generator, which does not fit this tutorial. You should run the following command to install the `@sap-cloud-sdk/generator` by your own.
+
+```
+npm install -g @sap-cloud-sdk/generator@^1.54.0
```

Comment:
This code is formatted as a note. Please either put inline code in single back-ticks or use the appropriate label.